docs/platform/howto/vnet-peering-azure.rst

```diff
@@ -108,7 +108,7 @@ as ``$user_app_secret`` below
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This can be found in the Azure portal in "Virtual networks" -> name of
-your network -> "Properties" -> "Resource ID", or using
+your network -> “JSON View” -> "Resource ID", or using
```

Comment:
As I understand, all the elements in quotation marks are visible in GUI so why not use bold (**) for them?